### PR TITLE
Refactored API endpoints

### PR DESCRIPTION
**Pull request by howardyang25**

### CODE REVIEW

File: client/components/Gallery/Gallery.jsx [hunk 1]
```diff
@@ -71,6 +70,7 @@ class Gallery extends React.Component {
     return (
       <div>
         {bigImage === undefined ? <div></div> : bigImage}
+        <title>Test Title</title>
```
Comment (howardyang25) [inline client/components/Gallery/Gallery.jsx]: I forgot to delete this test title

File: db/index.js [hunk 2]
```diff
@@ -37,16 +38,20 @@ const benuImageURLs = [
 ];
 
 benuImageURLs.forEach((url) => {
-  const image = new Image({ url });
+  const image = new Image({
+    url,
+    restaurantId: 1
```
Comment (ez-li) [inline db/index.js]: you don't need name, type?

File: db/index.js [hunk 3]
```diff
@@ -9,6 +9,7 @@ db.once('open', () => {
 });
 
 const imageSchema = new mongoose.Schema({
+  restaurantId: Number,
   url: String,
   name: String,
   type: String,
```
Comment (ez-li) [inline db/index.js]: what is type? not very descriptive

Comment (howardyang25) [reply db/index.js]: @ez-li Type is either from the restaurant, OpenTableDiner, or Foodspotting
Maybe rename to source?

File: db/index.js (hunk 2 above)
Comment (howardyang25) [reply db/index.js]: @ez-li This was based on an image from Benu, which had no extra information displayed aside from the image. When I seed my database later, I'll be sure to have a variety of types/names/photographers

**Pull request merged**